Maple2.Server.Game/Session/GameSession.cs

```diff
@@ -92,6 +92,10 @@ public sealed partial class GameSession : Core.Network.Session {
     public PartyManager Party { get; set; }
     public ConcurrentDictionary<int, GroupChatManager> GroupChats { get; set; }
 
+    public ConcurrentDictionary<ActorState, float> StateSyncDistanceTracking { get; set; }
+    public ConcurrentDictionary<ActorState, long> StateSyncTimeTracking { get; set; }
+    public long StateSyncTrackingTick { get; set; }
```

Comment:
I think it would be better suited in FieldPlayer

Reply:
Should we just move the entire OnStateSync function to FieldPlayer?

Reply:
honestly yeah I think it should be. All the values it's updating is solely on player